Google.PowerShell/Container/GkeClusterCmdlets.cs

```diff
@@ -161,4 +168,390 @@ private IEnumerable<Cluster> GetClustersByZone(string project, string zone)
             }
         }
     }
+
+    /// <summary>
+    /// <para type="synopsis">
+    /// Creates a Google Container Engine Node Config.
+    /// </para>
+    /// <para type="description">
+    /// Creates a Google Container Engine Node Config. The node config is used to configure various properties
+    /// of a node in a container cluster so you can use the object returned by the cmdlet in New-GkeCluster
```

Comment:
New-GkeCluster, or Add-GkeCluster? We do mix these two, but I do like the convention we have in places where `New` creates a new object, while `Add` actually puts things on GCP.